Jenkinsfile

```diff
@@ -441,6 +441,7 @@ pipeline {
       post {
         always {
           junit testResults: '**/target/failsafe-reports/*.xml'
+          findText regexp: ".*ERROR.*", fileSet: "ftests/**/log/server.log"
```

Comment:
You might want to have a look at the QA settings, for instance in this job: https://qa.nuxeo.org/jenkins/job/Deploy/job/IT-nuxeo-master-tests-cap-tomcat/configure:

![image](https://user-images.githubusercontent.com/1265714/78885473-fbe6ac80-7a5c-11ea-9b71-d1a468f828f4.png)

Reply:
Hi,

about the regular expression  '*ERROR.*'  ?

Reply:
Don't know, I was just pointing at the existing since we want the same behavior.